docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,165 @@
+# WebView
+
+You can access Web pages and Web content in your application. You can also use various features for Web browsing, such as loading and displaying Web pages and navigating through the browsing history.
+
+The main WebView features include:
+
+- Initializing and Creating WebView
+
+    You can [initialize a WebView](#initialize) and [create a WebView object](#initialize).
+
+- Managing Web page loading
```

Comment:
@younghajung 
We use Title case for all titles in Tizen docs, so request you to change this: 
- Manage Web Page Loading

Reply:
Done